libvncclient/listen.c

```diff
@@ -30,11 +30,12 @@
 #endif
 #include <sys/types.h>
 #ifdef WIN32
-#define close closesocket
```

Comment:
> Point taken. It leaves me wondering though why that error shows up.

Me too. @offtopic Mingw is very, very, very slow to compile libvncserver, it is tedious when we need to compile it from scratch. :-/

> Anyway, your patch passes Travis on Linux

It passes on Windows too. The travis errors are related to VS files that I never touched. :-)